docker_interop/app/src/main/resources/quorum_genesis_template.json

```diff
@@ -0,0 +1,38 @@
+{
+  "config": {
+    "chainId": 2017,
+    "homesteadBlock": 0,
+    "istanbul": {
+      "epoch": 30000,
+      "requestTimeout": %REQUEST_TIMEOUT%,
+      "blockPeriod": %BLOCK_PERIOD%,
```

Comment:
These options are I think can only be set from the command line looking at https://docs.goquorum.consensys.net/en/stable/Reference/Consensus/IBFTParameters/.

Reply:
introduced cli options to override these values when generating config files.